developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas

```diff
@@ -635,8 +635,8 @@ procedure TProjectFile.SaveState(node: IXMLNode);   // I4698
 procedure TProjectFile.UpdateID;
 begin
   if Assigned(FProject)
-    then FID := CleanID(MD5Hash(LowerCase( ExtractRelativePath(FProject.FileName, FFileName) )))   // I4709
-    else FID := CleanID(MD5Hash(LowerCase(FFileName)));
+    then FID := CleanID(THashMD5.GetHashString(LowerCase( ExtractRelativePath(FProject.FileName, FFileName) )))   // I4709
+    else FID := CleanID(THashMD5.GetHashString(LowerCase(FFileName)));
```

Comment:
I have tested and can confirm that `THashMD5.GetHashString` is a drop-in replacement for `MD5Hash`.

Reply:
So... what's our dependency for `THashMD5`, then?

Reply:
`System.Hash` unit -- included with Delphi.